### ISSUE TITLE
Full French Translations

### ISSUE THREAD

**Issue by GreatWizard** [astronautlevel2/Anemone3DS#46]
Hello from 🇫🇷,

I translate useful strings in French.
I remove `button` code from string because it's not supposed to change with locale.

```
"Installing a single theme..." : "Installation d'un seul thème..."
"Installing a shuffle theme..." : "Installation d'un thème aléatoire..."
"Installing BGM..." : "Installation de la musique..."
"NO THEMES FOUND" : "AUCUN THÈME TROUVÉ"
"Theme mode" : "Mode thème"
"By: " : "Par : "
"Install Shuffle Theme" : "Installer le thème aléatoire"
"Switch to Splashes" : "Passer aux lanceurs"
"Install Theme" : "Installer le thème"
"Queue Shuffle" : "Liste aléatoire"
"Install BGM" : "Installer musique"
"Preview Theme" : "Prévisualiser le thème"
"Scan QRCode" : "Scan QRCode"
"Installing a splash..." : "Installation d'un lanceur..."
"Uninstalling a splash..." : "Désinstallation d'un lanceur..."
"NO SPLASHES FOUND" : "AUCUN LANCEUR TROUVÉ"
"Splash mode" : "Mode lanceur"
"Install Splash" : "Installer lanceur"
"Switch to Themes" : "Passer aux thèmes"
"Delete current Splash" : "Supprimer le lanceur actuel"
```

 🍻

**Comment by Helloman892** [comment]
Thanks, as soon as @astronautlevel2 implements a framework, we'll get those in. Much appreciated!

**Comment by astronautlevel2** [comment]
Closed (#80)
